services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.db import transaction
+from django.db.models import QuerySet
+from django.contrib.auth import get_user_model
+
+from db.models import Order, Ticket
+
+
+User = get_user_model()

```

Comment:
This line is not necessary. You are already using `get_user_model()` to get the user model in your functions. This is a good practice as you are not hardcoding the user model.